src/dashboard/src/components/dashboard/Dashboard.tsx

```diff
@@ -96,17 +98,24 @@ export const Dashboard = () => {
     // When no filter is selected use all values available.
     if (
       isReadyTenants &&
-      !values.tenant &&
       isReadyOrganizations &&
-      !values.organization &&
       isReadyOperatingSystemItems &&
-      !values.operatingSystemItem &&
-      isReadyServerItems &&
-      !values.serverItem
+      isReadyServerItems
     ) {
-      updateDashboard({ reset: true });
+      if (
+        !values.tenant &&
+        !values.organization &&
+        !values.operatingSystemItem &&
+        !values.serverItem
+      ) {
+        updateDashboard({ reset: true });
+      } else if (init) {
+        updateDashboard({ applyFilter: true });
```

Comment:
It will apply the latest values to the dashboard when a user only belongs to one tenant and/or organization.

This does not fix the bug where the user should not see the All Organizations chart.